packages/ui/react-ui-theme/src/styles/fragments/focus.ts

```diff
@@ -5,6 +5,7 @@
 export const focusRing =
   'focus:outline-none focus-visible:z-[1] focus-visible:hover:outline-none dark:focus-visible:hover:outline-none focus-visible:ring-2 focus-visible:ring-offset-0 focus-visible:ring-primary-350 focus-visible:ring-offset-white dark:focus-visible:ring-primary-450 dark:focus-visible:ring-offset-black';
 
+// TODO(burdon): Requires container to have m-1.
```

Comment:
All rings render outside of the [border-box](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/The_box_model#parts_of_a_box) of the element they’re applied to, unless `ring-inset` is given. I think this comment is less of an actionable to-do and more of a reminder of how rings work.

Reply:
Did you notice I added m-1 to stacks since the ring was being cut-off.

Reply:
Yes, but that won’t strictly always be necessary except in that particular situation: where the element’s border box is the same size as its parent’s content box _and_ the parent does not have visible overflow.